### DISCUSSION TITLE
Can't get v1.0.0.0 to work!

### DISCUSSION THREAD

**Discussion by tig** [adamralph/minver#516] [Q&A] [answered]
I'm stumped. We've been using minver fine up to this point to manage pre-releases of `Terminal.Gui`. https://github.com/migueldeicaza/gui.cs

I'm now prepping for our first real Release which will be v1.0.0.0.

```
> git tag
v0.1
v0.2
v0.3
v0.4
v0.5
v0.90.0
v0.90.2
v0.90.3
v1.0.0-beta.10
v1.0.0-beta.11
v1.0.0-beta.6
v1.0.0-beta.7
v1.0.0-beta.8
v1.0.0-pre.1
v1.0.0-pre.2
v1.0.0-pre.3
v1.0.0-pre.4
v1.0.0-pre.5
v1.0.0-pre.6
v1.0.0-pre.7
v1.0.0-pre.8
v1.0.0-rc.1
v1.0.0-rc.2
v1.0.0-rc.5
v1.0.0.0
```

You would think doing `dotnet build -c Release` would use the `v1.0.0.0` tag, but it's stuck on `v1.0.0-rc.51:

```
> dotnet build -c Release
Microsoft (R) Build Engine version 16.9.0-preview-21076-28+8ed151254 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

...

  Successfully created package 'C:\Users\charlie\s\gui.cs\Terminal.Gui\bin\Release\Terminal.Gui.1.0.0-rc.5.nupkg'.
  Successfully created package 'C:\Users\charlie\s\gui.cs\Terminal.Gui\bin\Release\Terminal.Gui.1.0.0-rc.5.snupkg'.
```

I've tried deleting all tags for kicks. After doing this, even if the ONLY tag is `v1.0.0.0` minver continues to use `v1.0.0-alpha.xxx`!

I must be missing something simple.

Please help!

**Comment by bording** [comment, marked as answer]
@tig I suspect the problem you're having is that MinVer supports SemVer 2.0 versions, and `1.0.0.0` is not a valid SemVer version. You need to tag with `v1.0.0` instead, and it should work.

**Reply by adamralph** [reply]
@tig and if you [switch logging to detailed](https://github.com/adamralph/minver#can-I-get-log-output-to-see-how-minver-calculates-the-version) you'll see that MinVer sees the `v1.0.0.0` tag and ignores it because, as @bording says, `1.0.0.0` is not a valid SemVer version.

If you want to use non-SemVer versions, then I'm afraid MinVer isn't the right package for you.

**Reply by tig** [reply]
Oh, I misunderstood SemVer (actually didn't dive deep enough). I've changed my tag to v1.0.0 and all is good! 

Thanks for the help!
